en/developer_guide/routing.rst

```diff
@@ -0,0 +1,110 @@
+Routing
+=======
+
+Base URL
+--------
+
+In Open Orchestra, the base url of a website is composed by the domain and eventually a language prefix.
```

Comment:
Eventually is not a transparent word. I guess you mean "possibly" or "potentially".